README.md

```diff
@@ -1,21 +1,140 @@
+[![](https://img.shields.io/twitter/follow/Soramitsu_co?label=Follow&style=social)](https://twitter.com/Soramitsu_co)
+
+Kagome is a [Polkadot Runtime Environment](https://github.com/w3f/polkadot-spec/tree/master/runtime-environment-spec) developed by [Soramitsu](https://soramitsu.co.jp/) add funded by Web3 Foundation [grant](https://github.com/w3f/Web3-collaboration/blob/master/grants/grants.md). 
+
+## Supported features
+* Extrinsic Api
+    * Receives extrinsics submitted over JSON-RPC, validates them and stores in transaction pool
+* Transaction pool
+    * Stores submitted transactions
+    * Provides ready transactions, i.e. transaction which pre-conditions defined by tags are satisfied
+* Authorship
+    * Creates a block from provided inherents and digest
+    * Interacts with Polkadot Runtime to initialize and finalize block
+* Runtime
+    * Uses [Binaryen](https://github.com/WebAssembly/binaryen) WASM interpreter to invoke Polkadot Runtime entries
+    * Current runtime entries include:
+        * BlockBuilder – checks inherents, applies extrinsics, derives inherent extrinsics, initializes and finalizes blocks
+        * Core – gets version of runtime, executes blocks, gets authorities
+        * Grandpa – gets grandpa authorities
+        * Metadata (not used)
+        * OffchainWorker (not used)
+        * ParachainHost (not used)
+        * TaggedTransactionQueue – validates transactions
+* Externals (aka Extensions)
+    * Exposes a set functions that Runtime needs for:
+        * Management of the content
+        * Memory allocation
+        * Cryptography
+* SCALE (polkadot-codec)
+    * Implements SCALE codec for data types serialization according to [spec](https://substrate.dev/docs/en/conceptual/core/codec).
+* Storage  
+    * Contains of key-value storage interfaces with LevelDB- and inmemory-based implementations
+    * Merkle patricia trie implementation, described in spec
+* Clock
+    * Gets current time for the peer
+* Consensus
+    * Babe block production
+    * Grandpa finalization
+* Crypto
+    * blake2
+    * ed25519
+    * keccak
+    * sha
+    * Schnorr's vrf and sr25519 (bindings over Web3’s [schnorrkel library](https://github.com/w3f/schnorrkel))
+    * twox
+* Networking
+    * Kagome uses [cpp-libp2p](https://github.com/soramitsu/libp2p) for peer-to-peer interactions
+    * Gossiper and Gossiper observer
+    * SyncClient and SyncServer 
+    
+You can find more information about the components by checking [reference documentation](https://kagome.netlify.com). 
+
+## Trying out Kagome node
+
+As of now there is not much things you can do with Kagome node. However, you can already execute a block production process.
+
+### Clone
+    
+To clone repository execute: 
+1. `git clone https://github.com/soramitsu/kagome`
+2. `cd kagome`
+
+
+### Build kagome node
+
+To build kagome node binary go to your build folder and assemble `kagome_full` binary using cmake:
+```cmake
+cd build
```

Comment:
```suggestion
mkdir build && cd build
```

Reply:
This unit is located before build section, so this may be appropriate

Reply:
mkdir -p may also be useful in case 'build/' does exist, but our README is not a shell scripting guide so we may as well do without it.